src/controllers/requestController.js

```diff
@@ -1,21 +1,43 @@
-/* eslint-disable no-await-in-loop */
 /* eslint-disable guard-for-in */
 /* eslint-disable no-restricted-syntax */
-/* eslint-disable max-len */
 import db from '../database/models';
 import sendResult from '../utils/sendResult';
 
 const Request = {
   async getRequest(req, res) {
     try {
-      const requests = await db.Requests.findAll({ where: { UserId: req.userId }, raw: true });
-      for (const index in requests) {
-        requests[index].Trips = await db.Trips.findAll({ where: { RequestId: requests[index].id }, raw: true, attributes: { exclude: ['RequestId'] } });
-      }
+      const requests = await db.Requests.findAll({
```

Comment:
@urbanishimwe  good job but I suggest make get request-for-user a middleware so that it is re-usable because it will be used multiple times and code climate will fail